specification/signalr/data-plane/AzureSignalR/v1/azuresignalr.json

```diff
@@ -0,0 +1,605 @@
+          },
+          {
+            "in": "query",
+            "name": "ttl",
+            "description": "Specifies the seconds that the user exists in the group. If not set, the user lives in the group forever.",
+            "type": "integer",
+            "format": "int32"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Success"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "signalr.v1"
+        ],
+        "summary": "Remove a user from the target group.",
+        "operationId": "ServiceApi_RemoveUserFromGroup",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "hub",
+            "description": "Target hub name, which should start with alphabetic characters and only contain alpha-numeric characters or underscore.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[A-Za-z][A-Za-z0-9_`,.[\\]]{0,127}$"
+          },
+          {
+            "in": "path",
+            "name": "group",
+            "description": "Target group name, which length should be greater than 0 and less than 1025.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "user",
+            "description": "Target user Id",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Success"
+          }
+        }
+      }
+    },
+    "/api/v1/hubs/{hub}/users/{user}/groups": {
+      "delete": {
+        "tags": [
+          "signalr.v1"
+        ],
+        "summary": "Remove a user from all groups.",
+        "operationId": "ServiceApi_RemoveUserFromAllGroups",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "hub",
+            "description": "Target hub name, which should start with alphabetic characters and only contain alpha-numeric characters or underscore.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[A-Za-z][A-Za-z0-9_`,.[\\]]{0,127}$"
+          },
+          {
+            "in": "path",
+            "name": "user",
+            "description": "Target user Id",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The user is deleted"
+          },
+          "202": {
+            "description": "The delete request is accepted and service is handling the request int the background"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PayloadMessage": {
+      "required": [
+        "target"
+      ],
+      "type": "object",
+      "properties": {
+        "target": {
+          "type": "string"
+        },
+        "arguments": {
+          "type": "array",
+          "items": {
+            "type": "object"
```

Comment:
Is there no known structure to the items in the array? In other words, they can be arbitrary key/value pairs (but not strings etc.)?

Reply:
Yes the items of `PayloadMessage` can be any object, there is no constraint on the object.

Reply:
So they could be strings? If so, `"type": "object"` should be removed since that implies that it has to be a mapping like object (key/value pairs)